packages/@aws-cdk/aws-sam/README.md

```diff
@@ -0,0 +1,5 @@
+## AWS Serverless Construct Library
```

Comment:
Rename the title to something like "AWS Serverless Application Model Construct Library"

Reply:
Also, would be nice to mention that people should generally use the regular Lambda/API constructs instead of this.